build-tools-internal/src/main/groovy/elasticsearch.stable-api.gradle

```diff
@@ -0,0 +1,51 @@
+import org.elasticsearch.gradle.Version
+import org.elasticsearch.gradle.VersionProperties
+import org.elasticsearch.gradle.internal.BwcVersions
+import org.elasticsearch.gradle.internal.JarApiComparisonTask
+import org.elasticsearch.gradle.internal.info.BuildParams
+
+import java.util.stream.Collectors
+
+import static java.util.Arrays.stream
+
+configurations {
+  stableApi
+}
+
+BuildParams.bwcVersions.allIndexCompatible.findAll({ it.onOrAfter(Version.fromString("8.5.0")) && it != VersionProperties.elasticsearchVersion }).each { bwcVersion ->
+  def baseName = "v${bwcVersion}"
+
+  BwcVersions.UnreleasedVersionInfo unreleasedVersion = BuildParams.bwcVersions.unreleasedInfo(bwcVersion)
+  Configuration stableApiConfiguration = configurations.create("stableApi${baseName}") {
+    // TODO: Temporary workaround for https://github.com/elastic/elasticsearch/issues/73433
```

Comment:
This issue was specific to the JDBC driver? Does this exist for these libraries too?

Reply:
Doesn't look like it. I'm removing it.